docs/installation/get-started.md

```diff
@@ -0,0 +1,90 @@
+# Get started
+
+Get one GOV.UK Frontend component working in your application, so you can test everything works correctly before you add more components or styles.
+
+You will need to do the following.
+
+1. Install GOV.UK Frontend.
+2. Add the HTML for a component to your application.
+3. Get the CSS working.
+4. Get the font and images working.
+5. Get the JavaScript working.
+
+## 1. Install
+
+[Install GOV.UK Frontend using npm](/docs/installation/installing-with-npm.md).
+
+If you've installed from `dist`, get started with [a basic page](/docs/installation/installing-from-dist.md#2-include-resources) instead.
```

Comment:
I'm not sure it's clear what this refers to. A user might look at this and wonder what `dist` is and whether it's something to do with their own set up and something they should know about, instead of realising that it's an alternative to the first sentence in this section.

We could say something like this (or make it clear in some other way that these are alternative options):

> If you've [installed with the compiled files](/docs/installation/installing-from-dist) instead of npm, get started with [a basic page](/docs/installation/installing-from-dist.md#2-include-resources) and skip step 3 in this guidance.

Reply:
Sorry @hannalaakso, I've just realised I didn't respond to your comments!

Definitely agree here (and as you saw it came up in the migration docs research too). It's something I'm planning to improve across our docs in Q4, so I'm tempted to leave this as is for now to avoid confusion by having inconsistency with how we talk about installing from dist/compiled files. But I'm wavering, so I may well use your suggestion while this is getting 2i'd.